### PR TITLE
Filter users by role

### PR DESCRIPTION
**Pull request by andreiuta97**
Added the feature to filter the users by role.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,32 +1,35 @@
...
+      "type": "vcs",
+      "url": "git@github.com:andreiuta97/codeHighlightLib.git"
+    }
+  ],
+  "require": {
+    "andreeasabadis/mvc-framework-skeleton": "dev-master",
+    "really-orm-skeleton": "dev-master",
+    "andreeasabadis/code-highlight-lib": "dev-master",
+    "ext-pdo": "*",
+    "php": "7.2.5"
```
Comment (calinpristavu) [inline composer.json]: `php` key should be first in `"require"`. after the php key, you add the extensions, and after everything else, you add the dependencies

**Review by calinpristavu — CHANGES_REQUESTED** [review]

File: src/Controller/UserController.php [hunk 2]
```diff
@@ -117,15 +117,19 @@ public function deleteUser(Request $request, array $requestAttributes): Response
```
Comment (victortodoran) [inline src/Controller/UserController.php:117]: From what I can see now you can filter users by role. As the number of filters grows(including the possibility of multiple filters at once) the code in this method will become tedious.
If isset($rA['some_filter]) x 5 do smth. 
In a future iteration I would extract the creation of $filters in a separate function. 
I would also consider mapping filters together inside $requestAttributes if  that were possible. 

This is not a change request. Just my 2 cents.

File: composer.json (hunk 1 above)
Comment (andreiuta97) [reply composer.json]: Done in master (I hope it's not a problem) :)

File: src/Controller/UserController.php (hunk 2 above)
Comment (andreiuta97) [reply src/Controller/UserController.php:117]: Noted! Thank you!

**Pull request merged**